guides/common/modules/proc_republishing-repository-and-content-view-metadata.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Republishing_Repository_and_Content_View-Metadata_{context}"]
+= Republishing Repository and Content View Metadata
```

Comment:
I am unsure about two procedures in one file. What do you think? @bangelic @adamlazik1

Reply:
Should we make republishing Content View metadata its own procedure? It'll be a very quick one but I'm ok with it if you think it makes more sense to have it as a separate procedure.

Reply:
> I am unsure about two procedures in one file. What do you think? @bangelic @adamlazik1

Yeah, that the other thing that I wanted to point out. Probably better to have that in a separate file.